src/components/content-tab-import-export.tsx

```diff
@@ -1,6 +1,132 @@
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf } from '@wordpress/i18n';
+import { Icon, download } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { useRef } from 'react';
+import { useConfirmationDialog } from '../hooks/use-confirmation-dialog';
+import { useDragAndDropFile } from '../hooks/use-drag-and-drop-file';
+import { useSiteDetails } from '../hooks/use-site-details';
+import { cx } from '../lib/cx';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ImportState } from '../lib/import-export/import/types';
+import Button from './button';
+import { ProgressBarWithAutoIncrement } from './progress-bar';
+
 interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
-export function ContentTabImportExport( _props: ContentTabImportExportProps ) {
-	return null;
+
+export function ContentTabImportExport( props: ContentTabImportExportProps ) {
+	const { __ } = useI18n();
+	const { importFile, updateSite, startServer } = useSiteDetails();
+	const importConfirmation = useConfirmationDialog( {
+		message: sprintf( __( 'Overwite %s?' ), props.selectedSite.name ),
+		checkboxLabel: __( 'Don’t show this message again' ),
+		detail: __( 'Importing a backup will replace the existing files and database for your site.' ),
+		confirmButtonLabel: __( 'Import' ),
+		localStorageKey: 'dontShowImportConfirmation',
+	} );
+
+	const { dropRef, isDraggingOver } = useDragAndDropFile< HTMLDivElement >( {
+		onFileDrop: ( file: File ) => {
+			importConfirmation( () => importFile( file, props.selectedSite ) );
+		},
+	} );
+	const inputFileRef = useRef< HTMLInputElement >( null );
+	const openFileSelector = async () => {
+		inputFileRef.current?.click();
+	};
+	const onFileSelected = async ( e: React.ChangeEvent< HTMLInputElement > ) => {
+		const file = e?.target?.files?.[ 0 ];
+		if ( ! file ) {
+			return;
+		}
+		importConfirmation( () => importFile( file, props.selectedSite ) );
+	};
+	const openSite = async () => {
+		if ( ! props.selectedSite.running ) {
+			await startServer( props.selectedSite.id );
+		}
+		getIpcApi().openSiteURL( props.selectedSite.id );
+	};
+	const clearImportState = () =>
+		updateSite( { ...props.selectedSite, importState: ImportState.Initial } );
+	return (
+		<div className="p-8 flex flex-col justify-between gap-8">
+			<div className="flex flex-col w-full">
+				<div className="a8c-subtitle-small mb-1">{ __( 'Import' ) }</div>
+				<div className="text-a8c-gray-70 a8c-body mb-4">
+					{ createInterpolateElement(
+						__( 'Import a Jetpack backup or a .sql database file. <a>Learn more.</a>' ),
+						{
+							a: (
+								<Button
+									variant="link"
+									onClick={ () =>
+										getIpcApi().openURL(
+											'https://developer.wordpress.com/docs/developer-tools/studio/'
+										)
```

Comment:
Nitpick: We already have this URL defined in a constant, so we could use take from there.

https://github.com/Automattic/studio/blob/c8b1d1520ab8847c6c6b5fc9827a0e5e2cb54ef0/src/constants.ts#L13

Reply:
Done! https://github.com/Automattic/studio/pull/369/commits/b9be0b1013bc7defef1d10bfafe3f68acab4a56f